README.md

```diff
@@ -95,3 +95,109 @@ func (as *ActionSuite) Test_TodosResource_Update() {
 	as.Equal("Learn Go", todo.Title)
 }
 ```
+
+## Fixtures (Test Data)
+
+Often it is useful to load a series of data into the database at the start of the test to make testing. For example, you need to have a user in the database to log a person into the application, or you need some data in the database to test destroying that data. Fixtures let us solve these problems easily.
+
+### Usage
+
+First you need to setup your test suite to use fixtures. You can do this by using `suite.NewActionWithFixtures` or `suite.NewModelWithFixtures` methods to create new test suites that take a `packr.Box` pointing to where the files for this suite live.
+
+```go
+package actions
+
+import (
+	"testing"
+
+	"github.com/gobuffalo/packr"
+	"github.com/gobuffalo/suite"
+)
+
+type ActionSuite struct {
+	*suite.Action
+}
+
+func Test_ActionSuite(t *testing.T) {
+	action, err := suite.NewActionWithFixtures(App(), packr.NewBox("../fixtures"))
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	as := &ActionSuite{
+		Action: action,
+	}
+	suite.Run(t, as)
+}
+```
+
+Once your suite is set up, you can create `N` numbers of `*.toml` files in the directory you've chosen for your fixtures, in this example, `../fixtures`.
+
+### Example Fixture File
+
+```toml
+[[scenario]]
+name = "lots of widgets"
+
+  [[scenario.table]]
+    name = "widgets"
+
+    [[scenario.table.row]]
```

Comment:
It would be great if we could do something like loops for rows

Reply:
it's plush, so you can! :)